docs/README.md

```diff
@@ -18,7 +18,7 @@ Alternatively use ``sudo`` with the ``pip`` command to install the packages in t
 Optional:  if you want to build the PDF version of the manual, install LaTeX:
 
     # on Ubuntu
-    $ sudo apt-get install texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended
+    $ sudo apt-get install texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended latexmk
```

Comment:
*blinks*  Why do we need latexmk now?

Reply:
eh, I tried building the pdf and it didn't work... found [this](https://github.com/sphinx-doc/sphinx/issues/3543) which indicates it requires latexmk now. installed that, and it worked